src/components/Notification/Notification.tsx

```diff
@@ -0,0 +1,23 @@
+import classNames from 'classnames';
+import { useState } from 'react';
+
+export const Notification: React.FC = () => {
+  const [onButtonDelete, setOnButtonDelete] = useState(true);
+
+  return (
+    <div className={classNames(
+      'notification is-danger is-light has-text-weight-normal',
+      { hidden: onButtonDelete === false },
+    )}
+    >
+      <button
+        type="button"
+        className="delete"
+        aria-label="Mute volume"
+        onClick={() => setOnButtonDelete(false)}
+      />
+
+      Unable to add a todo
```

Comment:
You do not add there, but only upload, the text must correspond to the actions
```suggestion
      Unable to load a todo
```